src/net.h

```diff
@@ -967,7 +980,9 @@ class CConnman
 
     // Network stats
     void RecordBytesRecv(uint64_t bytes);
+    void RecordBytesRecvByMsgType(mapMsgTypeSize map_bytes_per_msg_type);
```

Comment:
Should this be one function since they are called in the same place? Hold off because this may change for `getnetworkmsgstats`?